.travis.yml

```diff
@@ -18,17 +16,15 @@ jobs:
       before_script:
         - pip install --upgrade numpy
         - pip install pandas===0.23.4
-        - pip install scikit-learn==0.20.3 # last scikit-learn version to support python 2.7
+        - pip install --upgrade scikit-learn
         - pip install --upgrade dask
         - pip install --upgrade distributed
         - pip install --upgrade scipy
         - pip list
       script:
         - sed -i 's/\-n auto/\-n 2/g' setup.cfg
-        - "if [ $TRAVIS_PULL_REQUEST == false ] && ! [ $TRAVIS_BRANCH == 'master' ]; then py.test tests/units -n2; else python setup.py test; fi"
-      python: 2.7
-      after_success:
-        - coveralls
```

Comment:
do we still have a coverall somewhere now?

Reply:
I added this now.